### ISSUE TITLE
Failed to load plugin '@typescript-eslint'

### ISSUE THREAD

**Issue by 0x1af2aec8f957** [typescript-eslint/typescript-eslint#3858]
<details>
  <summary><b>.eslintrc.json</b></summary>
 <pre>
{
  "root": true,
  "env": {
    "node": true
  },
  "extends": [
    "airbnb-base",
    "plugin:vue/vue3-essential",
    "plugin:@typescript-eslint/recommended",
    "eslint:recommended",
    "@vue/typescript"
  ],
  "globals": {
    "Atomics": "readonly",
    "SharedArrayBuffer": "readonly",
    "lodash": true,
    "_": true,
    "moment": true,
    "systemCode": true,
    "systemConfig": true,
    "ActiveXObject": true
  },
  "parser": "vue-eslint-parser",
  "parserOptions": {
    "parser": "@typescript-eslint/parser",
    "sourceType": "module"
  },
  "plugins": [
    "vue",
    "@typescript-eslint"
  ],
  "rules": {
    "no-console": "warn",
    "no-debugger": "warn",
    "global-require": 0,
    "indent": [
      "error",
      4
    ],
    "array-bracket-spacing": [
      2,
      "never"
    ],
    "block-scoped-var": 0,
    "brace-style": [
      2,
      "1tbs",
      {
        "allowSingleLine": true
      }
    ],
    "camelcase": 2,
    "comma-dangle": [
      2,
      "never"
    ],
    "comma-spacing": [
      2,
      {
        "before": false,
        "after": true
      }
    ],
    "comma-style": [
      2,
      "last"
    ],
    "complexity": [
      2,
      40
    ],
    "computed-property-spacing": [
      2,
      "never"
    ],
    "default-case": 2,
    "dot-location": [
      2,
      "property"
    ],
    "dot-notation": [
      2,
      {
        "allowKeywords": true
      }
    ],
    "eol-last": 2,
    "eqeqeq": [
      2,
      "allow-null"
    ],
    "import/no-unresolved": [
      2,
      {
        "ignore": [
          ""
        ]
      }
    ],
    "import/extensions": [
      "error",
      {
        "js": "never",
        "vue": "never"
      }
    ],
    "max-len": [
      "error",
      {
        "code": 500
      }
    ],
    "import/no-extraneous-dependencies": [
      "error",
      {
        "devDependencies": true
      }
    ],
    "arrow-parens": "off",
    "no-param-reassign": [
      "error",
      {
        "props": false
      }
    ],
    "no-plusplus": 0,
    "linebreak-style": [
      0,
      "error",
      "windows"
    ],
    "no-underscore-dangle": [
      0,
      {
        "allow": [
          "_place"
        ]
      }
    ],
    "consistent-return": [
      0,
      {
        "treatUndefinedAsUnspecified": true
      }
    ],
    "no-unused-vars": "off",
    "one-var": [
      "error",
      {
        "var": "always",
        "let": "always"
      }
    ],
    "one-var-declaration-per-line": [
      "error",
      "initializations"
    ],
    "no-return-assign": [
      "error",
      "always"
    ],
    "no-prototype-builtins": "off",
    "no-shadow": "off",
    "prefer-destructuring": [
      "error",
      {
        "object": true,
        "array": false
      }
    ],
    "radix": [
      "error",
      "as-needed"
    ],
    "no-case-declarations": "off",
    "no-nested-ternary": "off",
    "no-lonely-if": "off",
    "operator-assignment": "off",
    "object-curly-newline": "off",
    "class-methods-use-this": "off",
    "@typescript-eslint/no-empty-function": "off",
    "@typescript-eslint/ban-ts-comment": "off",
    "@typescript-eslint/no-unused-vars": "off",
    "@typescript-eslint/no-explicit-any": "off",
    "@typescript-eslint/ban-types": "off",
    "@typescript-eslint/no-inferrable-types": "off",
    "@typescript-eslint/explicit-module-boundary-types": "off"
  }
}
</pre>
```
</details> 

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `4.31.0` |
| `@typescript-eslint/parser`        | `4.31.0` |
| `TypeScript`                       | `4.4.2` |
| `ESLint`                           | `8.0.0-beta.1` |
| `node`                             | `16.8.0` |
| `ts-node`                             | `10.2.1` |

**Question**

This problem occurred after I upgraded `eslint` to `8.0.0-beta.1`. There is no such problem in the configuration with `eslint` version of `7.x.x`.
```bash
ERROR in Failed to load plugin `@typescript-eslint` declared in `.eslintrc.json`: Class extends value undefined is not a constructor or null
Referenced from: /tmp/test-project/.eslintrc.json
```

**Comment by bradzacher** [comment]
#3738
Please use the issue search.
